project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module

```diff
@@ -569,6 +569,14 @@ function c4m_user_profile_form_user_register_form_submit($form, &$form_state) {
     )
   );
 
+  // Always send "welcome" mail for user created by admin.
+  $admin = user_access('administer users');
+  if ($admin) {
+    $account = user_load($form['#user']->uid);
```

Comment:
@anvmn 
Did you check if `$form['#user']` is already the object of the user ?
we might not need the `user_load()` in here.